stable/faucet/faucet/values.yaml

```
@@ -48,3 +48,8 @@ Configuration: |-
           - rule:
               actions:
                   allow: True
+### SLATE-START ###
+SLATE:
+  Instance:
+    ID: "default-id"
```

Let's change this default
```suggestion
    ID: "untagged"
```